src/CollapseEmbeds/hider.tsx

```diff
@@ -36,8 +36,10 @@ export const Hider = ({placeholders, type, children, id}: HiderProps): JSX.Eleme
     }, [id]);
 
     const [shown, setShown] = React.useState(() => {
-        if (!id) return !Settings.current.hideByDefault;
-        return !Settings.current.collapsedStates[id]?.collapsed;
+        if (id && Settings.current.collapsedStates[id]) {
+            return !Settings.current.collapsedStates[id].collapsed;
+        }
+        return !Settings.current.hideByDefault;
```

Comment:
If the saved state in `collapsedStates` is `false` this will end up returning `hideByDefault` instead of `false`. Either use the `in` operator or `hasOwnProperty`.